modules/config/pages/eap-config.adoc

```diff
@@ -1,11 +1,159 @@
+
+. **log**: The default location for the server log files.
+
+. **tmp**:  For temporary files, like the shared-key mechanism for authenticating local users to the server's command-line interface (CLI).
+
+
+== JBoss EAP Base Directory
+
+* Users can separate the configuration and data directories from the JBoss EAP 8 installation directory, aiding easier upgrades.
+* This allows the binaries to be installed once and run multiple times on the same machine.
+* Two command-line properties enable this:
+
+. **jboss.home.dir**: Installation directory for JBoss EAP binaries.
+. **jboss.server.base.dir**: Base directory for server configuration files.
+
+* You can override runtime properties at the command line by using the `-D` option when starting the server.
+* For instance, when starting the server with standalone.sh:
++
+[subs="+quotes,+macros"]
+----
+$ ./standalone.sh -Djboss.server.base.dir=/path/to/base/directory -Djboss.home.dir=/path/to/home/directory
+----
+
+== Lab: Run a standalone JBoss EAP 8 server using a custom location for the server base directory.
+
+*Outcome*: In this lab, you should be able to run a standalone JBoss EAP 8 server using a custom location for the server base directory.
+
+* Create a custom location for the server base directory using below command.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo mkdir -p /opt/standalone-running
+----
+
+* Copy the configuration, deployments, and lib directories to the new location.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ cd /opt/EAP-8.0.0/standalone/
+[vagrant@server standalone]$ sudo cp -r configuration deployments lib /opt/standalone-running
+----
+
+* Modify the standalone server port offset.
+
+. To verify that the JBoss EAP server is utilizing the new, copied configuration files, update the port offset. You can run multiple instances of EAP, by using the port offset to avoid port conflicts.
+
+. Update the new configuration file by editing the standalone.xml file to verify that JBoss EAP is using the configuration files in the new directory. Open the `/opt/standalone-running/configuration/standalone.xml` file with a text editor.
+
+. In the standard-sockets socket binding group, change the default value of the port-offset attribute from 0 to 10000. This adds 10,000 to every port number used in the standard-sockets binding group.
++
+[subs="+quotes,+macros"]
+----
+[vagrant@server ~]$ sudo vi /opt/standalone-running/configuration/standalone.xml
+...
+<socket-binding-group name="standard-sockets" default-interface="public" port-offset="${jboss.socket.binding.port-offset:10000}">
+...
+----
+
+Save your changes to the standalone.xml file, and close the text editor.
+
+* Run and test the JBoss EAP Server.
+
+. Run the following command to start the JBoss EAP server by using the standalone.sh script in the original JBoss EAP installation, but using the new configuration files:
```

Comment:
Pass another -D flag, -Djboss.socket.binding.port-offset:10000

Reply:
check AD248 for the correct flag

Reply:
updated the changes